### PR TITLE
Fix uptime gauge [API-1382]

### PR DESCRIPTION
**Pull request by utku-caglayan**

### CODE REVIEW

File: internal/stats/stats_service.go [hunk 1]
```diff
@@ -257,7 +264,11 @@ func (g runtimeGauges) updateNumCPU(bt *binTextStats) {
 }
 
 func (g runtimeGauges) updateUptime(bt *binTextStats) {
-	if uptime, err := host.Uptime(); err != nil {
+	if g.proc == nil {
+		// gopsutil.process is not registered, skip gauge
+		return
+	}
+	if uptime, err := g.proc.CreateTime(); err != nil {
```
Comment (yuce) [inline internal/stats/stats_service.go:271]: The uptime would be calculated as: `time.Now() - g.proc.CreateTime()`.

**Review by yuce — CHANGES_REQUESTED** [review]
Just uptime calculation is not correct.

File: internal/stats/stats_service.go (hunk 1 above)
Comment (utku-caglayan) [reply internal/stats/stats_service.go]: Thanks for catching that. Copied time.Time to ms utility from Nearcache PR. This way we won't make duplication when they are merged.

https://github.com/hazelcast/hazelcast-go-client/pull/808/commits/94f37a2947d456aedaab0b20093381cb89c23057

**Review by srknzl — DISMISSED** [review]

File: internal/stats/stats_service.go (hunk 1 above)
Comment (yuce) [reply internal/stats/stats_service.go]: OK, but I've removed that stuff from NC, since we will upgrade to Go 1.17 that one is not necessary anymore.

**Review by yuce — APPROVED** [review]

**Pull request merged**